plugins/modules/packaging/os/pacman_key.py

```diff
@@ -331,6 +322,7 @@ def main():
         ),
         supports_check_mode=True,
         mutually_exclusive=(('data', 'file', 'url', 'keyserver'),),
+        required_one_of=[('data', 'file', 'url', 'keyserver'),],
```

Comment:
The previous code requires one of those **only** when `state` is `present`. This code imposes that on all states.

Reply:
Yes, I just realized it was my own comment :)